app/src/main/java/de/danoeh/antennapod/activity/MainActivity.java

```diff
@@ -224,6 +224,9 @@ private void checkFirstLaunch() {
         if (prefs.getBoolean(PREF_IS_FIRST_LAUNCH, true)) {
             new Handler().postDelayed(() -> drawerLayout.openDrawer(navDrawer), 1500);
 
+            // for backward compatibility, we only change defaults for fresh installs
+            UserPreferences.setUpdateInterval(12);
```

Comment:
Do you always do exactly what you are told? ;)
Just thought it would not really hurt to refresh at least twice a day.